liquibase-core/src/test/groovy/liquibase/changelog/ChangeSetTest.groovy

```diff
@@ -647,4 +647,22 @@ public class ChangeSetTest extends Specification {
         changeSet.execute(databaseChangeLog, new MockDatabase()) == ChangeSet.ExecType.EXECUTED
     }
 
+    def "execute does not execute shell command against when adding MDC"() {
```

Comment:
Let me make sure I understand this test. Here is how I'm reading it: 

* Create a changelog with a `executeShellCommandChange` that creates a file when executed.
* Call ChangeSet.execute for this changelog.
* ChangeSet.execute will _not_ run the `executeShellCommandChange` and _not_ create the file because the `allowNonExecuteMode=true` for the `ExecuteShellCommandChange`.

The test passes if the file is not created.
The test fails if the file is created.

Then, in cleanup, the test attempts to delete the file. Since the file will not exist (in the passing scenario), the File.delete will return false, but we don't care because that result is not used anywhere. 

Am I reading this correctly, @wwillard7800 ?